### DISCUSSION TITLE
bouwen

### DISCUSSION THREAD

**Discussion by ju5tu5** [fdnd/docs.fdnd.nl#62] [Boksa]
Hier iets vertellen over code-literacy. Met name de opbouw/volgorde/structuur van HTML/CSS/JS bestanden.

**Comment by KoopReynders** [comment]
Performance opnemen. 

Onderdelen van dit artikel hebben we met de studenten behandeld: 
https://www.smashingmagazine.com/2021/01/front-end-performance-2021-free-pdf-checklist/

Perceived performance:

Cheating The UX When There Is Nothing More To Optimize
https://stephaniewalter.design/blog/pixel-pioneers-cheating-the-ux-when-there-is-nothing-more-to-optimize/

**Comment by KoopReynders** [comment]
Bronnen voor UX & Pleasurable interfaces
> https://blog.logrocket.com/ux-design/designing-button-states/

Delight users through CSS Animation Techniques
https://stephaniewalter.design/blog/delighting-users-css-animation-techniques-conveyux-slides-demos/

Applying Disney's basic principles of animation to UI design
https://dribbble.com/stories/2020/07/27/disney-principles-of-animation-ui-interactions

Choose Your Ease - Interaction Prop tips
https://aerotwist.com/tutorials/protip-choose-your-ease/

Designing button states: Tutorial and best practices
https://blog.logrocket.com/ux-design/designing-button-states/
